tests/lib/eslint-tester.js

```diff
@@ -329,6 +329,27 @@ describe("ESLintTester", function() {
         });
     });
 
+    it("should pass-through the parser to the rule", function() {
```

Comment:
Ok for the devDependency.

Just to be sure, you're suggesting to add a `getParser()` method on `context` in ESLint ?